packages/run-protocol/test/vaultFactory/test-vaultFactory.js

```diff
@@ -1695,12 +1686,22 @@ test('mutable liquidity triggers and interest', async t => {
 
   aliceUpdate = await E(aliceNotifier).getUpdateSince(aliceUpdate.updateCount);
   t.deepEqual(aliceUpdate.value.debtSnapshot.debt, aliceRunDebtLevel);
+  trace(t, 'alice reduce collateral');
 
-  await manualTimer.tick();
+  await E(priceAuthority).setPrice(makeRatio(7n, runBrand, 1n, aethBrand));
+  trace(t, 'changed price to 7');
+
+  // await manualTimer.tick();
   // price levels changed and interest was charged.
 
   // expect Alice to be liquidated because her collateral is too low.
   aliceUpdate = await E(aliceNotifier).getUpdateSince(aliceUpdate.updateCount);
+  trace(t, 'alice liquidating?', aliceUpdate.value.vaultState);
+  t.is(aliceUpdate.value.vaultState, Phase.LIQUIDATING);
+
+  bobUpdate = await E(bobNotifier).getUpdateSince();
```

Comment:
safer way to iterate updates:
```suggestion
  bobUpdate = await E(bobNotifier).getUpdateSince(bobUpdate.count);
```

Reply:
These are actually different: with no argument, it asks for the current state. By passing the argument, it says "block until the state actually changes".  Since I'm trying to verify that the state does NOT change, it needs to not provide the argument. NOTE that there was a test bug that it was not verifying that things did NOT change. Thus when the interlock bug allowed multiple outstanding changes, bob's vault liquidated too early but this test didn't notice.

Reply:
> These are actually different: with no argument, it asks for the current state. By passing the argument, it says "block until the state actually changes".

Yeah, that's what I consider an improvement.

> Since I'm trying to verify that the state does NOT change

But that's brittle because you don't know whether it _will_ change. 

> it needs to not provide the argument

Not necessarily; it could provide `updateCount - 1`. Then that would help express the intent to the reader.

Reply:
That doesn't address it either. We need to include settling.